.pre-commit-config.yaml

```diff
@@ -0,0 +1,10 @@
+# See https://pre-commit.com for more information
+# See https://pre-commit.com/hooks.html for more hooks
+repos:
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v2.4.0
+    hooks:
+    -   id: trailing-whitespace
+    -   id: end-of-file-fixer
+    -   id: check-yaml
+    -   id: check-added-large-files
```

Comment:
Also needs flake8 config here I think as per https://github.com/pre-commit/pre-commit/blob/master/.pre-commit-config.yaml#L14.

Reply:
not applicable since we're just going to try with flake8 for now...